composer.json

```diff
@@ -78,8 +78,8 @@
         "illuminate/view": "self.version"
     },
     "require-dev": {
-        "aws/aws-sdk-php": "^3.155",
         "doctrine/dbal": "^2.6|^3.0",
+        "aws/aws-sdk-php": "^3.155",
```

Comment:
You've moved this dependency, please move it back.

Reply:
@driesvints Check now.